experimental/grpc_attestation/proto/grpc_attestation.proto

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.grpc_attestation;
+
+import "google/protobuf/empty.proto";
+
+message GetAttestedDataRequest {
+  oneof request_type {
+    AttestationRequest attestation_request = 1;
+    bytes data = 2;
+  }
+}
+
+message AttestationRequest {
+  bytes public_key = 1;
+}
+
+message GetAttestedDataResponse {
+  oneof response_type {
+    AttestationResponse attestation_response = 1;
+    bytes data = 2;
+  }
+}
+
+message AttestationResponse {
+  bytes public_key = 1;
+  bytes attestation_info = 2;
+}
+
+service GrpcAttestation {
+  rpc GetAttestedData(stream GetAttestedDataRequest) returns (stream GetAttestedDataResponse);
```

Comment:
Thanks. I did not know about those naming conventions. To better follow the naming convention, perhaps we could drop `Attested` from the method name and the request and response types, since the service name already includes `Attestation`. I'd suggest the following but I don't insist on it, and I am not entirely happy with it either :) 
`rpc SendInvocation(stream InvocationRequest) returns (stream InvocationResponse);`